pkg/controllers/workgenerator/controller_test.go

```diff
@@ -677,7 +678,6 @@ func TestSetBindingStatus(t *testing.T) {
 					},
 				},
 			},
-			maxFailedResourcePlacementLimit: 2,
```

Comment:
looks like we don't test when it's 2 anymore?

Reply:
The default is 100. if the failed number < limit, 2 or 100 does not matter.